examples/chat/module/rust/build.rs

```diff
@@ -17,6 +17,6 @@
 fn main() {
     oak_utils::compile_protos(
         &["../../proto/chat.proto"],
-        &["../../proto", "../../third_party"],
+        &["../../proto", "../../third_party", "../../../../"],
```

Comment:
This is needed because we are importing oak protos: https://github.com/project-oak/oak/blob/daccc103aaba3be637e30ebdafee97ea27a52cf3/examples/chat/proto/chat.proto#L22-L23.

I think you are right that the `third_party` is non-sense, I'll remove that.